UPGRADE-6.0.md

```diff
@@ -23,7 +23,8 @@ Console
 -------
 
  * `Command::setHidden()` has a default value (`true`) for `$hidden` parameter
- * Remove `Helper::strlen()`, use `Helper::width()` instead.
+ * Removed `Helper::strlen()`, use `Helper::width()` instead.
```

Comment:
revert the preview syntax was right: https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry